acp/main_module.php

```diff
@@ -0,0 +1,208 @@
+<?php
+/**
+ *
+ * phpBB Media Embed PlugIn extension for the phpBB Forum Software package.
+ *
+ * @copyright (c) 2016 phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ */
+
+namespace phpbb\mediaembed\acp;
+
+/**
+ * phpBB Media Embed Plugin ACP module.
+ */
+class main_module
+{
+	/** @var \phpbb\cache\driver\driver_interface $cache */
+	protected $cache;
+
+	/** @var \phpbb\config\config $config */
+	protected $config;
+
+	/** @var \phpbb\config\db_text $config_text */
+	protected $config_text;
+
+	/** @var \Symfony\Component\DependencyInjection\ContainerInterface $container */
+	protected $container;
+
+	/** @var \phpbb\request\request $request */
+	protected $request;
+
+	/** @var \phpbb\template\template $template */
+	protected $template;
+
+	/** @var \phpbb\language\language $language */
+	protected $language;
+
+	/** @var \phpbb\log\log $log */
+	protected $log;
+
+	/** @var \phpbb\user */
+	protected $user;
+
+	/** @var array $enabled_sites */
+	protected $enabled_sites;
+
+	/** @var string $form_key */
+	protected $form_key;
+
+	/** @var string $page_title */
+	public $page_title;
+
+	/** @var string $tpl_name */
+	public $tpl_name;
+
+	/** @var string $u_action */
+	public $u_action;
+
+	/**
+	 * Constructor
+	 */
+	public function __construct()
+	{
+		global $phpbb_container;
```

Comment:
*cries a little* Please re-write the ACP module system to use services and tags and I'll love you forever.